tests/sentry/event_manager/test_hierarchical_hashes.py

```diff
@@ -68,15 +69,22 @@ def inner(last_frame):
         )
 
         with task_runner():
-            return _save_aggregate(
-                evt,
-                hashes=hashes,
-                release=None,
-                metadata={},
-                received_timestamp=0,
-                migrate_off_hierarchical=False,
-                **group_creation_kwargs,
-            )
+            with patch(
+                "sentry.event_manager.get_hash_values",
+                return_value=(hashes, hashes, hashes),
+            ):
+                with patch(
+                    "sentry.event_manager._get_group_creation_kwargs",
+                    return_value=group_creation_kwargs,
+                ):
+                    with patch("sentry.event_manager._materialize_metadata_many"):
+                        return _save_aggregate(
+                            evt,
+                            job={"event_metadata": {}},
+                            release=None,
+                            received_timestamp=0,
+                            metric_tags={},
+                        )
```

Comment:
LGTM, as long as these methods have coverage elsewhere.

Reply:
Yeah. This is just compensating for the fact that it used to be that those methods lived outside of `_save_aggregate`, and therefore calling `_save_aggregate` directly bypassed them entirely. Once they moved into `_save_aggregate`, suddenly they stopped being bypassed and started causing problems (because they would require much more extensive mocking than happens in this test). 

This restores the bypass by either no-oping them entirely or changing their return values to be the same thing which was getting passed to `_save_aggregate`. (So, in the case of `get_hash_values`, for instance, `hashes` used to be passed in, and was hardcoded in the test. Now those hashes are calculated at the beginning of `_save_aggregate`, so this makes it so the same hardcoded value is used.) Does that make sense?